indexer/packages/postgres/src/stores/pnl-ticks-table.ts

```diff
@@ -537,7 +537,8 @@ export async function getLatestPnlTick(
       pnl_ticks
     WHERE
       "subaccountId" in (${subaccountIds.map((id: string) => { return `'${id}'`; }).join(',')}) AND
-      "blockTime" <= '${beforeOrAt.toUTC().toISO()}'::timestamp
+      "blockTime" <= '${beforeOrAt.toUTC().toISO()}'::timestamp AND
+      "blockTime" >= '${beforeOrAt.toUTC().minus({ hours: 4 }).toISO()}'::timestamp
```

Comment:
curious why 4 hours? 1 hour with some buffer should be sufficient?

Reply:
Just to catch any case where pnl ticks don't get produced for more than an hour + any edge cases with times on the hour. 1 hour and 4 hour doesn't affect the query much.